operations/availabilityblocks.md

```diff
@@ -66,231 +62,266 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
-| `Extent` | [Availability block extent](#availability-block-extent) | required | Extent of data to be returned, e.g. it is possible to specify that related service orders (for example reservations) are returned. |
-
-#### Availability block state
-
-* `Confirmed` - The block deducts availability and can have reservations assigned.
-* `Optional` - The block deducts availability and cannot have reservations assigned.
-* `Inquired` - The block does not deduct availability and cannot have reservations assigned \(waitlist\).
-* `Released` - The block deducts availability, but only for reservations assigned to the block before release. Remaining availability is returned back to general availability \(waitlist\).
-* `Canceled` - The block does not deduct availability and cannot have reservations assigned \(waitlist\). 
+| `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the Enterprises. If not specified, the operation returns data for all enterprises within scope of the Access Token. |
+| `Extent` | [Availability block extent](availabilityblocks.md#availability-block-extent) | required | Extent of data to be returned, e.g. it is possible to specify that related service orders (for example reservations) are returned. |
+| `ServiceIds` | array of string | optional, max 1000 items | Unique identifiers of the `Services` to which `Availability blocks` are assigned. |
+| `AvailabilityBlockIds` | array of string | optional, max 1000 items | Unique identifiers of the requested `Availability blocks`. |
+| `CreatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the availability blocks were created. |
+| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the `Availability blocks` were updated. |
+| `CollidingUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the `Availability blocks` are active. |
+| `ReleasedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the `Availability blocks`are released. |
+| `ExternalIdentifiers` | array of string | optional, max 1000 items | Identifiers of `Availability blocks` from external systems. |
+| `States` | array of [Availability block state](availabilityblocks.md#availability-block-state) | optional | States the availability blocks should be in. |
+| `ActivityStates` | array of string | optional | Whether to return only active, only deleted or both records. |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of data returned and optional Cursor for the starting point of data. |
 
 #### Availability block extent
+Extent of data to be returned, e.g. it is possible to specify that related service orders (for example reservations) are returned.
```

Comment:
issue: Duplicated description with property description, I think you'll have to set it to empty string on the respective DTO.